src/main/java/mate/academy/bookstorev2/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.bookstorev2.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import mate.academy.bookstorev2.validation.FieldMatch;
+
+@Getter
+@Setter
+@FieldMatch(firstString = "password",
+        seconfString = "repeatPassword",
+        message = "The password fields must match")
+public class UserRegistrationRequestDto {
+    @NotBlank
+    private String password;
+    @NotBlank
+    @Email(message = "Email is not valid",
```

Comment:
Suggestion , test this pattern if it works correctly. Also, you can create your custom validation like for passwords, but with pattern